contracts/Oracle.sol

```diff
@@ -95,14 +95,15 @@ contract Oracle is Ownable {
         bytes32 commitmentIndex,
         uint256 blockNumber,
         string calldata blockBuilderName,
-        bool isSlash
+        bool isSlash,
+        uint256 residualBidAfterDecay
     ) external onlyOwner {
         // Check graffiti against registered builder IDs
         address builder = blockBuilderNameToAddress[blockBuilderName];
-        
+        require(residualBidAfterDecay <= 100, "Residual bid after decay cannot be greater than 100 percent");
```

Comment:
Ah seems like this is a percentage then? `bidPercentAfterDecay` or similar might be a more appropriate name

Reply:
Agreed